tools/ci_build/build.py

```diff
@@ -700,8 +700,9 @@ def main():
             if is_windows() or not os.path.exists(onnx_test_data_dir):
                 onnx_test_data_dir = os.path.join(source_dir, "cmake", "external", "onnx", "onnx", "backend", "test", "data")
             if args.use_tensorrt:
-              # Disable onnx unit tests for TensorRT because many tests are not supported yet
-              run_onnx_tests(build_dir, configs, '', 'tensorrt', False, 1)
+              # Disable some onnx unit tests that TensorRT parser doesn't supported yet
+              onnx_test_data_dir = os.path.join(source_dir, "cmake", "external", "onnx", "onnx", "backend", "test", "data", "simple")
```

Comment:
what is the purpose of line 704?  (test_data_dir points to simple sub directory)

Reply:
To enable onnx unit tests. For debug mode those tests must be enabled.

Reply:
so only the tests in the 'simple' subdir currently passes and nothing else?

Reply:
Yes. Not all tests in other folders are supported by TensorRT